<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdraw</class>
 <widget class="QWidget" name="withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnWithdraw20">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw40">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>40€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw60">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw100">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw200">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>200€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw500">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdrawCustom">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Muu summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountInfo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoMessage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWithdrawCustom">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>14</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
